requirements-test.txt

```diff
@@ -1,4 +0,0 @@
-black
```

Comment:
These versions should be moved into test-requirements.txt

Reply:
removed the file